src/views/ContinuationOut.vue

```diff
@@ -254,25 +295,29 @@ import { useBusinessStore, useConfigurationStore, useRootStore } from '@/stores'
 
 @Component({
   components: {
+    BusinessNameForeign,
     Certify,
     ConfirmDialog,
     CourtOrderPoa,
     DetailComment,
     DocumentDelivery,
-    PaymentErrorDialog,
+    EffectiveDate,
+    ForeignJurisdiction,
     ResumeErrorDialog,
     SaveErrorDialog,
-    SbcFeeSummary,
-    StaffPaymentDialog
+    SbcFeeSummary
   }
 })
 export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
```

Comment:
I didn't look deeply, but does this filing have a check that the user is staff? If not, please add one as a safety check.

Reply:
Thanks!